### ISSUE TITLE
Some questions about "MetaBatchNorm2d" module

### ISSUE THREAD

**Issue by lizijue** [g3956/PTKP#4]
Dear author:

Thank you for your excellent work and for sharing the code with us.

Firstly, I apologize for bothering you, but I have encountered some confusion while reading the code. Specifically, I would like to know the motivation behind manually defining the “MetaBatchNorm2d” class, as I did not notice any difference between it and the “nn.BatchNorm2d” class.

Also, why did you remove the "bias" properties of both “Shared BN” and “TSBN” from the computational graph via “xx.bias.requires_grad_(False)”? Do we not use the “bias” parameter to move the Gaussian distribution to an ideal position?

I would appreciate it if you could provide some guidance on these questions.

Sincerely,

Zijue

**Comment by g3956** [comment]
Thanks for your interest in our work.

For the first issue, there is no difference between “MetaBatchNorm2d” and “nn.BatchNorm2d”. I tried other methods which rely on “MetaBatchNorm2d” during this project but finally abandon them and the class is left.

For the second issue, it is a general operation for person re-identification tasks, I just followed the previous code. But I did not test the performance of “xx.bias.requires_grad_(True)”. Maybe you can have a try.
